cmake/Modules/FindComputeCpp.cmake

```diff
@@ -193,6 +193,21 @@ function(__build_spir targetName sourceFile binaryDir fileCounter)
     endforeach()
   endif()
 
+  # Obtain language standard of the file
+  set(device_compiler_cxx_standard)
+  get_target_property(targetCxxStandard ${targetName} CXX_STANDARD)
+  if (targetCxxStandard MATCHES 17)
+    set(device_compiler_cxx_standard "-std=c++1z")
+  elseif (targetCxxStandard MATCHES 14)
+    set(device_compiler_cxx_standard "-std=c++14")
+  elseif (targetCxxStandard MATCHES 11)
+    set(device_compiler_cxx_standard "-std=c++11")
+  elseif (targetCxxStandard MATCHES 98)
+    message(FATAL_ERROR "SYCL implementations cannot be compiled using C++98")
```

Comment:
What about C++03 and some other weird versions?
I guess we do not care... :-)

Reply:
These are the ones CMake knows.

https://cmake.org/cmake/help/v3.9/prop_tgt/CXX_STANDARD.html

I'm not discriminating anything here... ;)